Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'http://rubygems.org'
+
+gem "rspec"
```

Comment:
I use rspec for pretty much everything but have no problem with another testing framework if you have strong opinions.